common/libs/wallet/base.js

```diff
@@ -4,4 +4,8 @@ export default class BaseWallet {
   getAddress(): string {
     throw 'Implement me';
   }
+
+  getNakedAddress(): string {
+    return this.getAddress().replace('0x', '').toLowerCase();
```

Comment:
I'm not sure what the standard is on address formats, but is a naked address always lower cased? I only ask because I know there's a reason why they are upper or lower case.